exporters/committime/collector_containerimage.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/env python3
```

Comment:
I do not like the idea of adding new exporter without any new tests

At least some usage tests like these https://github.com/dora-metrics/pelorus/blob/master/exporters/tests/test_committime_exporter_app.py#LL172C1-L227C84

Reply:
Added unit tests, not covering 100% but the rest of the functions are common openshift tested elsewhere.